templates/single.html

```diff
@@ -48,4 +48,4 @@
 <!-- /wp:group --></div>
 <!-- /wp:group -->
 
-<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
```

Comment:
Not sure what happened here, this change is unrelated, I copied the whole file from trunk to solve the merge conflict.